scripts/charge-custom-fields-upload/main.py

```diff
@@ -0,0 +1,63 @@
+import csv
+import requests
+import concurrent.futures
+
+API_URL = "https://api.metronome.com"
+TOKEN = ""
+INTERNAL_ID_CSV = "internal_ids_example.csv"
+INTERNAL_ID_CSV_CHARGE_NAME_HEADER = "Billable metric name"
+INTERNAL_ID_CSV_INTERNAL_ID_HEADER = "NetSuite Internal ID"
+CHARGES_CSV = "charges_example.csv"
+CHARGES_CSV_CHARGE_NAME_HEADER = "charge_name"
+CHARGES_CSV_CHARGE_ID_HEADER = "charge_id"
+CUSTOM_FIELD_NAME = "netsuite_internal_item_id"
+
```

Comment:
Should we make these arguments for the script?

Reply:
we could!  I didn't want to go too far since I think it might be worth having a product convo about this at some point -- ideally the API/UI is good enough that clients can do this themselves